### ISSUE TITLE
Creator: double click at instance switch to publish tab

### ISSUE THREAD

**Issue by jakubjezek001** [ynput/OpenPype#4567]
**Is your feature request related to a problem? Please describe.**
At the moment after a instance had been created user is pushed to either click at arrow showing at right frame or click to **publish** tab. It feels nonintuitive sometimes and someone would want to double click at instance to show instance properties in **publish** tab.

**Describe the solution you'd like**
Add action at created instances so user can double click and it will show him instance attributes at **publish** tab. 

**Additional context**
![image](https://user-images.githubusercontent.com/40640033/222694147-5288a320-b0af-4855-be93-6aac51d28998.png)

**Comment by BigRoy** [comment]
I wonder whether this might be confusing again once selection of subsets might not be visible to begin with after this issue is solved: https://github.com/ynput/OpenPype/issues/4316

Or does double clicking then still feel like a logical way?

**Comment by jakubjezek001** [comment]
> Or does double clicking then still feel like a logical way?

Yes, it still does. It has nothing to do with the design. This feels more intuitive to be able to click on the instance and the properties should show. But don't get me wrong on this as we are in match in some points you are having in the other issue (I will elaborate at it).
